src/lookup.py

```diff
@@ -463,6 +463,21 @@ def __init__(self, node: ET.Element):
 
                     self.elements.append(element)
 
+            def hasOverlappingElements(self):
+                for element in self.elements:
+                    for element2 in self.elements:
+                        if (
+                            element != element2
+                            and element.offset <= element2.offset
+                            and (element.offset + element.length) > element2.offset
+                        ):
+                            printf(
+                                f"Element {element.name} (Length: {element.length}, Offset: {element.offset}) conflicts with {element2.name} (Length: {element.length}, Offset: {element.offset})"
+                            )
+                            return True
+
+                return False
```

Comment:
I would be good to also validate whether there are gaps in the bitfield as specified and log a warning